### ISSUE TITLE
Apply interceptors to a custom Axios instance

### ISSUE THREAD

**Issue by kHorozhanov** [zhaosiyang/axios-observable#8]
I'm got this error when trying to apply interceptors to a custom Axios instance - **Property 'interceptors' is a static member of type 'Axios'**.

```javascript 
const instance = Axios.create();
instance.interceptors.request.use(function () {/*...*/});```

**Comment by zhaosiyang** [comment]
@kHorozhanov Thanks for reporting this issue. I am working on fixing it asap.

**Comment by zhaosiyang** [comment]
fixed by #9 and published v1.1.2
